seatunnel-api/src/main/java/org/apache/seatunnel/api/table/converter/BasicDataConverter.java

```diff
@@ -74,6 +74,8 @@ default Object convert(SeaTunnelDataType typeDefine, Object value) {
                 return convertTime(value);
             case TIMESTAMP:
                 return convertLocalDateTime(value);
+            case TIMESTAMP_TZ:
+                return convertOffsetDateTime(value);
```

Comment:
I am very interested in this. When can we make use of it?

Reply:
example：

mysql

datetime -> seatunnel timestamp
timestamp -> seatunnel timestamp_tz

Reply:
ok, thanks for answering